service-front/module/Application/src/Model/Service/Payment/Helper/LpaIdHelper.php

```diff
@@ -1,9 +1,11 @@
 <?php
+
 namespace Application\Model\Service\Payment\Helper;
+
 class LpaIdHelper
 {
-    const LPA_ID_LENGTH = 11;
-    
+    public const LPA_ID_LENGTH = 11;
```

Comment:
Just wondered why that needed to change to public

Reply:
This was a precommit fix. But with PSR any `const` or `var` on a class should have their scope set.